prisma/schema.prisma

```diff
@@ -112,3 +113,30 @@ model Attendees {
   attendeeId Int
   attendee   Users      @relation(fields: [attendeeId], references: [id])
 }
+
+enum CalendarType {
+  GOOGLECAL
+  OUTLOOKCAL
+}
+
+enum TokenType {
+  BEARER
+}
+
+model AccessToken {
+  id              Int          @id @default(autoincrement())
+  user            Users        @relation(fields: [userId], references: [id])
+  userId          Int
+  associatedEmail String
+  tokenType       TokenType
+  calendarId      String
+  calendarType    CalendarType
+  scope           String
+  expiry          Int
+  accessToken     String
+  refreshToken    String
+  createdAt       DateTime     @default(now())
+  updatedAt       DateTime     @updatedAt
+
+  @@unique([userId, calendarType, calendarId], name: "AccessToken_constraint")
```

Comment:
Just `userId` and `calendarId` will also be unique, right? Any benefit of adding calendarType here?

Reply:
Not really just an additional constraint.

Reply:
Yes. We shouldnt add it unnecessarily...
https://stackoverflow.com/questions/3860222/does-adding-a-unique-constraint-slow-down-things

Reply:
Thanks for sharing this, will keep in mind from next time.